misc/attrib.pm

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/perl
+#
+=head1  NAME
+
+    attrib.pm - two subs to handle ARQ and RRQ message attributes for HPT perl
+    by Stas Mishchenkov 2:460/58.
+
+=head1 SYNOPSIS
+
+   use attrib;
+   
+   sub filter()
+   {
+      is_rrq( $attr );
+   }
+   
+   sub route()
+   {
+      is_arq( $attr, $route );
+   }
+   
+   $attr - message attributes as defined by HPT perl.
+   $route - message routing as defined by HPT perl.
+
+=head1 DESCRIPTION
+
+   This subs to handle ARQ and RRQ message attributes for HPT perl designed
+   according to FTS-0001.016.
+   Insert into HPT configuration file:
```

Comment:
Since the subroutines will be present inside filter.pl, there should be a line like the following one in your HPT configuration file: